src/brsTypes/components/RoAssociativeArray.ts

```diff
@@ -125,7 +125,7 @@ export class RoAssociativeArray extends BrsComponent implements BrsValue, BrsIte
             returns: ValueKind.Boolean,
         },
         impl: (interpreter: Interpreter, str: BrsString) => {
-            let deleted = this.elements.delete(str.value);
+            let deleted = this.elements.delete(str.value.toLowerCase());
```

Comment:
Added this because I noticed it as I was testing. We're storing all the fields as lowercase, so we need to access them that way as well.